src/SprykerEco/Shared/Payone/Transfer/payone.transfer.xml

```diff
@@ -318,6 +318,7 @@
         <property name="payonePostfinanceEfinanceOnlineTransfer" type="PayonePaymentOnlinetransfer"/>
         <property name="payonePostfinanceCardOnlineTransfer" type="PayonePaymentOnlinetransfer"/>
         <property name="payonePrzelewy24OnlineTransfer" type="PayonePaymentOnlinetransfer"/>
+        <property name="payoneBancontactOnlineTransfer" type="PayonePaymentOnlinetransfer"/>
```

Comment:
`PayonePaymentOnlineTransfer`?

Reply:
`payoneBancontactOnlineTransfer` follows other transfers names.

Reply:
typo in type :)